fuel-types/src/bytes.rs

```diff
@@ -68,9 +40,23 @@ pub unsafe fn from_slice_unchecked<const N: usize>(buf: &[u8]) -> [u8; N] {
 #[allow(clippy::erasing_op)]
 #[allow(clippy::identity_op)]
 fn padded_len_to_fit_word_len() {
-    assert_eq!(WORD_SIZE * 0, padded_len(&[]));
-    assert_eq!(WORD_SIZE * 1, padded_len(&[0]));
-    assert_eq!(WORD_SIZE * 1, padded_len(&[0; WORD_SIZE]));
-    assert_eq!(WORD_SIZE * 2, padded_len(&[0; WORD_SIZE + 1]));
-    assert_eq!(WORD_SIZE * 2, padded_len(&[0; WORD_SIZE * 2]));
+    assert_eq!(Some(WORD_SIZE * 0), padded_len(&[]));
+    assert_eq!(Some(WORD_SIZE * 1), padded_len(&[0]));
+    assert_eq!(Some(WORD_SIZE * 1), padded_len(&[0; WORD_SIZE]));
+    assert_eq!(Some(WORD_SIZE * 2), padded_len(&[0; WORD_SIZE + 1]));
+    assert_eq!(Some(WORD_SIZE * 2), padded_len(&[0; WORD_SIZE * 2]));
+}
+
+#[test]
+fn padded_len_value() {
```

Comment:
Can we name this something like `padded_len_usize__gives_values_in_increments_of_8` or something like that. To say what the tested behavior is.